src/components/Person/Person.jsx

```diff
@@ -1 +1,35 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+export const Person = ({ person }) => (
+  <section className="Person">
+    <h2 className="Person__name">
+      My name is
+      {' '}
+      {person.name}
```

Comment:
Let's destructure the person
const { name, age. isMarried ... } = person;
```suggestion
      My name is {name}
```

Reply:
Fixed